openadapt/ml_models/huggingface_models/create_modal_image.py

```diff
@@ -0,0 +1,71 @@
+import modal
```

Comment:
What do you think about moving this `utils.py`?

Also consider creating a `utils` directory and refactoring the contents, and moving this `utils/modal.py`.